SharedSupport/Default Bundles/MD5 Hash.spBundle/command.plist

```diff
@@ -37,5 +37,7 @@ print hash
 	<string>8CAA33B3-D168-47FC-A9BE-EA5A8576E380</string>
 	<key>isDefaultBundle</key>
 	<true/>
+	<key>migratedToNewShortcut</key>
+	<true/>
```

Comment:
Some of these bundles have a migratedToNewShortcut but not a <key>newShortcut</key>. Is this expected?

Reply:
Yes, they are defaults, but don't have a keyEquivalent.